### PR TITLE
Update AWS SDK version, sbt version, add test, fix log lines

### PR DESCRIPTION
**Pull request by realstraw**

### CODE REVIEW

**Comment by realstraw** [comment]
I cannot reproduce https://github.com/krux/starport/issues/43 , so just added some test @talbright

File: build.sbt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-val awsSdkVersion = "1.11.486"
+val awsSdkVersion = "1.11.618"
```
Comment (talbright) [inline build.sbt]: should we use ivy syntax to specify 1.11.* so it automatically gets bumped?

Comment (realstraw) [reply build.sbt]: I think we should do that in libraries, but I'd prefer not do it in starport where we ship a binary runnable, it's better for making static and deterministic versions.

Comment (talbright) [reply build.sbt]: Gotcha. Think we debated this before, makes perfect sense.

**Pull request merged**